server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/QualityProfileDao.java

```diff
@@ -149,26 +150,38 @@ public QualityProfileDto selectParentById(DbSession session, int childId) {
     return descendants;
   }
 
+  /**
+   * @deprecated provide organization
+   */
+  @Deprecated
   @CheckForNull
   public QualityProfileDto selectByNameAndLanguage(String name, String language, DbSession session) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)